t/client.t

```diff
@@ -61,9 +62,42 @@ GET /t
 --- no_error_log
 [error]
 
+=== TEST 2: simple ssl fetch
```

Comment:
style: need three blanks line before this line, and you are recommended to use the `reindex` tool to handle it automatically.
https://github.com/openresty/openresty-devel-utils#reindex

Reply:
spacing fixed!